connector/connector.js

```diff
@@ -70,3 +70,35 @@ connector.createMiddleware = function(phase, data, cb) {
   middlewareDef,
   cb);
 };
+
+connector.createModel = function(id, data, cb) {
```

Comment:
agreed , changed to index.js. here the memory connector is accessed via app.datasources, so there is no module.exports. we are not creating, we are customizing an existing object.